process.go

```diff
@@ -37,18 +37,19 @@ func process(fs fileSystem, filePaths []string) (poFile, error) {
 		if !isSupportedExt(filepath.Ext(filePath)) {
 			continue
 		}
-
-		content, err := fs.readFile(filePath)
-		if err != nil {
-			return poFile, err
+		f, e := os.Open(filePath)
+		if e != nil {
+			panic(e)
```

Comment:
I think it's better to keep the `panic()`s in the main and use plain errors throughout the code base.
```suggestion
			return poFile, err
```